cmd/kube-apiserver/app/encryption/encryption.go

```diff
@@ -100,17 +123,18 @@ func ParseEncryptionConfiguration(f io.Reader) (map[schema.GroupResource]value.T
 	return result, nil
 }
 
-// GetPrefixTransformer constructs and returns the appropriate prefix transformers for the passed resource using its configuration
-func GetPrefixTransformers(config *ResourceConfig) ([]value.PrefixTransformer, error) {
```

Comment:
This function and others which are relying on a generic resource config must remain in the generic apiserver package to make it easy/possible for any apiserver to encrypt or decrypt their data in a consistent way.  This is not kube-apiserver specific.